P5/Source/Specs/teidata.gender.xml

```diff
@@ -0,0 +1,19 @@
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+-->
+<?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<dataSpec xmlns="http://www.tei-c.org/ns/1.0" xmlns:tei="http://www.tei-c.org/ns/1.0" module="tei"
+  ident="teidata.gender">
+  <desc versionDate="2022-05-17" xml:lang="en">defines the range of attribute values used to
+    represent the gender of a person, persona, or character.</desc>
+  <content>
+    <dataRef key="teidata.enumerated"/>
+  </content>
+  <remarks versionDate="2022-08-27" xml:lang="en">
+    <p>Values for attributes using this datatype may be defined locally by a project, or they may refer to an external standard.</p>
+    <p>Values for this datatype should not be used to encode morphological gender (cf. <gi>gen</gi>,
+        <att>msd</att> as defined in att.linguistic, and <ptr target="#DITPFO"/>).</p>
```

Comment:
Should be 
~~~xml
<ident type="class">att.linguistic</ident>
~~~
(As much as I do not _like_ that encoding (because I think it is a name, not an identifier), it is what we use.)

Reply:
Fixed with https://github.com/TEIC/TEI/pull/2299/commits/1bd0f6a5ffa6712e20c0e545881b5755eff0073b